x/ccv/provider/keeper/msg_server.go

```diff
@@ -333,11 +413,24 @@ func (k msgServer) CreateConsumer(goCtx context.Context, msg *types.MsgCreateCon
 		}
 	}
 
-	ctx.EventManager().EmitEvents(sdk.Events{
-		sdk.NewEvent(ccvtypes.EventTypeConsumerCreation,
-			sdk.NewAttribute(ccvtypes.AttributeConsumerID, consumerId),
+	// add Phase event attribute
+	phase := k.GetConsumerPhase(ctx, consumerId)
+	eventAttributes = append(eventAttributes, sdk.NewAttribute(types.AttributeConsumerSpawnTime, phase.String()))
```

Comment:
Shouldn't this be the `spawnTime` instead of `phase.String`?
Also `AttributeConsumerPhase ` does not seem to be used.

Reply:
It should be AttributeConsumerPhase. I'll fix it.